source/reference/about-mongosync.txt

```diff
@@ -0,0 +1,131 @@
+clusters before it can start to sync data. For more information, see 
+:ref:`c2c-connecting`.
+
+When ``mongosync`` connects to the source and destination clusters, it is in the 
+:ref:`IDLE <c2c-state-idle>` state.  
+
+During Sync 
+~~~~~~~~~~~~
+
+The :ref:`start <c2c-api-start>` endpoint begins the 
+:ref:`RUNNING <c2c-state-running>` state. 
+
+If you start a new sync, ``mongosync`` syncs data to the destination cluster. 
+If you make subsequent writes to the source cluster, ``mongosync`` applies them 
+to the destination cluster.  
+
+For every collection on the source, ``mongosync`` creates a corresponding 
+collection on the destination. 
+
+``mongosync`` temporarily alters some collection characteristics during 
+the ``RUNNING`` state and changes these back to their source 
+cluster characteristics during the ``COMMIT`` state. For more information, see
+:ref:`Commit Alters <c2c-commit-alters>`.
+
+Collection Copy
+'''''''''''''''
+
+Collection copy is the initial full data transfer that copies each collection 
+from the source cluster to the destination cluster. ``mongosync`` performs 
+collection copy by splitting each source collection into partitions, gathering 
+all partitions from all collections into one list, and copying all partitions 
+to their corresponding destination collections.
+
+Change Event Application
+'''''''''''''''''''''''''
+
+Change event application (CEA) is an ongoing process that continuously updates 
+the destination cluster when there are changes to the source cluster. If there 
+are multiple ``mongosync`` instances, all the ``mongosync`` instances finish 
+collection copy before entering CEA.
+
+During CEA, ``mongosync`` receives :ref:`change stream <changeStreams>` events 
+from the source cluster, converts these events into writes, and applies these 
+writes on the destination cluster. CEA ensures both :ref:`CRUD <crud>` and 
+:ref:`DDL <c2c-mongosync-ddl>` operations are handled accurately.
+
+Pausing Sync
+~~~~~~~~~~~~~
+
+The :ref:`pause <c2c-api-pause>` endpoint begins the 
+:ref:`PAUSED <c2c-state-paused>` state. To use the ``pause`` endpoint, 
+``mongosync`` must be in the ``RUNNING`` state.
+
+During the ``PAUSED`` state, the sync process is paused.
+
+The :ref:`resume <c2c-api-resume>` endpoint resumes a paused synchronization 
+session and returns ``mongosync`` to the ``RUNNING`` state. To use the 
+``resume`` endpoint, ``mongosync`` must be in the ``PAUSED`` state.
+
+Committing Sync
+~~~~~~~~~~~~~~~~
+
+The :ref:`commit <c2c-api-commit>` endpoint begins the 
+:ref:`COMMITTING <c2c-state-committing>` state. To use the ``commit`` endpoint, 
+``mongosync`` must be in the ``RUNNING`` state. 
+
+During the ``COMMITTING`` state, ``mongosync`` finalizes the sync between the 
+source cluster and the destination cluster by stopping continuous sync 
+between them and restoring collection characteristics that ``mongosync`` 
+temporarily alters during synchronization. 
```

Comment:
[suggestion] This sentence is really long. Can you split it into multiple sentences?

Reply:
Split into two sentences